jetty-util/src/main/java/org/eclipse/jetty/util/thread/ThreadPoolBudget.java

```diff
@@ -142,10 +142,10 @@ public void check() throws IllegalStateException
         int maximum = pool.getMaxThreads();
         int actual = maximum - required;
 
-        if (actual <= 0)
+        if (actual < 0)
         {
             infoOnLeases();
-            throw new IllegalStateException(String.format("Insufficient configured threads: required=%d < max=%d for %s", required, maximum, pool));
+            throw new IllegalStateException(String.format("Insufficient configured threads: max=%d < required=%d for %s", required, maximum, pool));
```

Comment:
You've swap the order of the format string to have max before required, but the arg order is still required then maximum

Reply:
Fixed. Thank you.